main.js

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env node
+
+const env = require('./.env');
+const DB = process.argv[2] || env.from.DB;
+const USER = process.argv[3] || env.from.USER;
+const PASS = process.argv[4] || env.from.PASS;
+const HOST = process.argv[5] || env.from.HOST;
+
+if (!DB || !USER || !PASS || !HOST) {
+  console.error('Some configuration is missing, please set the .env file with the configuration or pass it through parameters');
+  process.exit(1);
+}
+
+const fs = require('fs');
+const path = require('path');
+const mysql = require('./config/mysql.js');
+const dateFormat = require('./utils/dateFormat.js');
+const migrations = require('./utils/migrations.js');
+const writeLog = require('./utils/writeLog.js');
+const chalk = require('chalk');
+const start = Date.now();
+
+const createMigrationFile = (file_data, table_name, db_name, isView = false) => {
+  return new Promise((resolve, reject) => {
+    // Creating migrations' directory
+    const pathMigrations = './migrations';
+    fs.mkdir(path.resolve(__dirname, pathMigrations), { recursive: true }, (err) => {
+      if (err) {
+        writeLog(err);
```

Comment:
Too strange to a log method to throw errors and stop execution. Also the logging here is quite unusual, as it writes to a file and gives the user no feedback. We should use a standard logging library here and the user must get feedback about the error in the console too.